libfwupdplugin/meson.build

```diff
@@ -360,7 +375,8 @@ if get_option('tests')
     ],
     link_with: [
       fwupd,
-      fwupdplugin
+      fwupdplugin,
```

Comment:
there's a lot more places that link with `fwupdplugin`, this might need to be done more ubiquitously up front to make sure we don't have weird bugs later as more is converted to rust or goes in as rust.

Reply:
Ohh, it's linked *into* fwupdplugin -- so anything linking fwupdplugin gets fwupdpluginrs "for free"